reflex/app.py

```diff
@@ -1503,6 +1508,65 @@ def _write_stateful_pages_marker(self):
             with stateful_pages_marker.open("w") as f:
                 json.dump(list(self._stateful_pages), f)
 
+    def add_active_connections_endpoint(self):
+        """Add an endpoint to the app that returns the active connections."""
+        if not self._api:
+            return
+
+        async def active_connections(_request: Request) -> Response:
+            if not self.event_namespace:
+                return JSONResponse({})
+            return JSONResponse(self.event_namespace.token_to_sid)
```

Comment:
this is just a tad bit crazy, because the client_token is basically the keys to the kingdom for a user's state and it's assumed to be private.

we need to log a big huge fat warning when this setting is enabled letting the user know that their app privacy is completely compromised.

with these two endpoints it's theoretically possible to steal cookies or other private data from any session.